python/run_plotXSTemplates.py

```diff
@@ -0,0 +1,413 @@
+'''
+void plotXSTemplates(const string & _year_="2018", const string & _obsTag_="pT4l", const string & _obsTagLeg_="p_{T}",const string & _obsBins_="|0|10|20|30|45|60|80|120|200|13000|", const int & _nBins_=9)
+'''
+#########################
+# original plotter:
+#  https://github.com/vukasinmilosevic/Fiducial_XS/blob/master/FidXS_13TeV_2018_80X_preM19_newBinning_preApp/templates_Freeze/plotXSTemplates.C
+########################
+
+import ROOT
+import sys, os, pwd
+from subprocess import *
+import optparse, shlex, re
+import math
+import time
+from decimal import *
+import json
+from collections import OrderedDict as od
+import seaborn as sns
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from Utils import processCmd
+
+from tdrStyle import *
+from read_bins import *
+import yaml
+
+def parseOptions():
+
+    global opt, args, runAllSteps
+
+    usage = ('usage: %prog [options]\n'
+             + '%prog -h for help')
+    parser = optparse.OptionParser(usage)
+
+    # input options
+    parser.add_option('',   '--theoryMass',dest='THEORYMASS',    type='string',default='125.38',   help='Mass value for theory prediction')
+    #parser.add_option('',   '--obsName',  dest='OBSNAME',  type='string',default='pT4l',   help='Name of the observable, supported: "inclusive", "pT4l", "eta4l", "massZ2", "nJets"')
+    parser.add_option('',   '--obsName',  dest='OBSNAME',  type='string',default='mass4l',   help='Name of the observable, supported: "inclusive", "pT4l", "eta4l", "massZ2", "nJets"')
+    #parser.add_option('',   '--obsBins',  dest='OBSBINS',  type='string',default='|0|10|20|30|45|60|80|120|200|13000|',   help='Bin boundaries for the diff. measurement separated by "|", e.g. as "|0|50|100|", use the defalut if empty string')
+    parser.add_option('',   '--unblind', action='store_true', dest='UNBLIND', default=False, help='Use real data')
+    parser.add_option('',   '--inYAMLFile', dest='inYAMLFile', type='string', default="Inputs/observables_list.yml", help='Input YAML file having observable names and bin information')
+    parser.add_option('',   '--year',  dest='YEAR',  type='string',default='2018',   help='Era to analyze, e.g. 2016, 2017 or 2018')
```

Comment:
Add the three more options as in this macro: 

https://github.com/ram1123/Fiducial_XS/blob/844721c58f073cbd58f6a2c731382206fa53f694/python/plotDifferentialBins.py#L38-L40

This will prevent the plots from popping up. If it pops up, it takes too much time to run.